csharp/src/Ice/IPEndpoint.cs

```diff
@@ -50,6 +46,8 @@ option switch
 
         public override bool IsLocal(Endpoint endpoint)
         {
+            // TODO: revisit, is connection ID gone?
```

Comment:
It's not gone yet. Getting rid of it in the source is easy but it will require to revisit a bunch of tests that rely on it.